server/game/setupcardaction.js

```diff
@@ -18,6 +18,7 @@ class SetupCardAction extends BaseAbility {
 
     meetsRequirements(context) {
         return (
+            context.game.readyToStart &&
```

Comment:
This variable doesn't seem to be set outside of tests??? There's a Player.readyToStart but not at the game level.